includes/class-wc-amazon-payments-advanced-api.php

```diff
@@ -1959,4 +1961,35 @@ public static function send_email_notification( $subject, $message, $recipient )
 		$mailer->send( $recipient, strip_tags( $subject ), $message );
 	}
 
+	/**
+	 * Return if subscription is shippable
+	 *
+	 * @param WC_Order $order Order object.
+	 *
+	 * @return boolean
+	 */
+	public function maybe_subscription_is_shippable( WC_Order $order ) {
```

Comment:
Sorry about Travis not running the unit tests, it should be working now.

It might be a PHP version thing, but I get an error when running the unit tests (and I imagine at runtime as well):

```
Non-static method WC_Amazon_Payments_Advanced_API::maybe_subscription_is_shippable() should not be called statically
```

So I think this function should be declared as static? We'll need to add the new parameter to the unit tests as well.